.github/workflows/ci.yml

```
@@ -297,12 +298,20 @@ jobs:
           name: output
           path: _output/**
 
-      - name: Login to Docker
+      - name: Login to DockerHub
         uses: docker/login-action@v1
         if: env.DOCKER_USR != ''
         with:
           username: ${{ secrets.DOCKER_USR }}
           password: ${{ secrets.DOCKER_PSW }}
+      
+      - name: Login to Upbound Registry
+        uses: docker/login-action@v1
+        if: env.UPBOUND_ROBOT_USR != ''
```

```suggestion
        if: github.ref == 'refs/heads/master' && env.UPBOUND_ROBOT_USR != ''
```

It seems like we use that login only in that condition.

---

@muvaf this is also used in the `Publish`` step, I'll add a check to make sure it is set there, which will make this change not necessary 👍🏻